### PR TITLE
Add new action "gravityflowformconnector_processed_{stepType}"

### PR DESCRIPTION
**Pull request by spivurno**
to allow doing something after a new entry has been created.

### CODE REVIEW

File: includes/class-step-new-entry.php [hunk 1]
```diff
@@ -208,6 +208,19 @@ public function process_local_action() {
...
+			 * Do something after the New Entry step has been processed.
+			 *
+			 * @since x.x.x
+			 *
+			 * @param int                          $entry_id The newly created entry ID.
+			 * @param array                        $entry    The entry for which the step was processed.
+			 * @param array                        $form     The form for which the entry was submitted.
+			 * @param \Gravity_Flow_Step_New_Entry $this     The current instance of the Gravity_Flow_Step_New_Entry class.
+			 */
+			do_action( 'gravityflowformconnector_processed_' . $this->get_type(), $entry_id, $entry, $form, $this );
```
Comment (Idealien) [inline includes/class-step-new-entry.php:222]: Based on [our naming conventions](https://github.com/gravityflow/gravityflow/wiki/Naming-conventions-for-hooks) would go with gravityflowformconnector_post_' . $this->get_type().

File: includes/class-step-new-entry.php [hunk 2]
```diff
@@ -208,6 +208,19 @@ public function process_local_action() {
 					$this->maybe_store_new_entry_id( $entry_id );
 				}
 			}
+
+			/**
+			 * Do something after the New Entry step has been processed.
```
Comment (Idealien) [inline includes/class-step-new-entry.php:213]: Fires after the New Entry FC step has been processed.

File: includes/class-step-new-entry.php [hunk 3]
```diff
@@ -208,6 +208,19 @@ public function process_local_action() {
...
+			 *
+			 * @since x.x.x
```
[the remark this answers is not in the document]
Comment (spivurno) [reply includes/class-step-new-entry.php]: Done.

File: includes/class-step-new-entry.php (hunk 2 above)
Comment (spivurno) [reply includes/class-step-new-entry.php]: Done.

File: includes/class-step-new-entry.php (hunk 1 above)
Comment (spivurno) [reply includes/class-step-new-entry.php]: Done.

**Comment by spivurno** [comment]
All requests have been completed. Let know if you need anything else from me, @Idealien. 🙂

**Comment by spivurno** [comment]
Hey @Idealien, it was just pointed out to me that this will generate a notice if the `$new_entry` is empty since the `$entry_id` variable will not have been declared. My instinct is to just set `$entry_id = null;` before the condition so that it is set either way and the hook would always process but I'm not sure if you guys have a precedent that should be followed here?

**Pull request merged**